input/templates/my-account-edit-address.hbs

```diff
@@ -22,19 +22,19 @@
               </div>
               <div id="my-account-desktop-content" class="col-sm-9">
                 <div class="address-book-content edit-address-book-content">
-                  <form id="form-edit-my-address-book" action="{{@root.meta._links.editMyAddressBook.href}}" method="POST">
+                  <form id="form-edit-my-address-book" action="{{@root.meta._links.myAddressBookEditAddressSubmit.href}}" method="POST">
                     <input type="hidden" name="csrfToken" value="{{@root.meta.csrfToken}}"/>
-                    <input type="hidden" name="id" value="{{content.address.id}}"/>
+                    <input type="hidden" name="id" value="{{content.addressForm.id}}"/>
                     <div class="row">
                       <div class="col-sm-12">
                         <div class="address-book-title hidden-xs">
-                          {{i18n "my-account:addressBook.title"}} > {{i18n "my-account:addressBook.normalAddress.title"}} > {{i18n "my-account:buttons.edit"}}
+                          {{i18n "my-account:addressBook.title"}}
                         </div>
                       </div>
                     </div>
                     <div class="row">
                       <div class="col-sm-12">
-                        {{> myaccount/edit-address-form-elements form=addressForm.billingAddress suffix="Billing"}}
+                        {{> myaccount/my-address-book-address-form form=content.addressForm}}
```

Comment:
Besides the rename of the partial template (same reason as before: I'm using this for the new address too), it is important to see that before this form page was only processing billing addresses. As it is a page with only one address, which you can decide whether to make it default for shipping and/or billing, it can be perfectly called simply "address", without any qualification. So the JSON `content` will only have an `addressForm` for this form, nothing else.